VultisigApp/VultisigApp/Views/Keysign/KeysignMessageConfirmView.swift

```diff
@@ -26,7 +26,10 @@ struct KeysignMessageConfirmView: View {
                 
                 HStack {
                     Spacer()
-                    if (viewModel.keysignPayload?.coin.chainType == .EVM && blowfishResponse != nil) {
+                    if (
+                        (viewModel.keysignPayload?.coin.chainType == .EVM || viewModel.keysignPayload?.coin.chainType == .Solana) &&
```

Comment:
Views should not contain logic. Only read properties from viewModels

Reply:
Moved to the view model.